packages/docs/package.json

```diff
@@ -1,72 +1,55 @@
 {
```

Comment:
Please revert the changes to this file. This is the `package.json` for the `docs` package, not for the top-level monorepo.

Reply:
Sorry copied back the correct package.json for the docs